doc/api/fs.md

```diff
@@ -378,7 +378,9 @@ fs.appendFile('message.txt', 'data to append', 'utf8', callback);
 
 Any specified file descriptor has to have been opened for appending.
 
-_Note: Specified file descriptors will not be closed automatically._
+_Note: If a file descriptor is specified as the 'file', it will not be closed automatically._
```

Comment:
Backticks instead of single quotes around `file` here and in the other notes.

In other words:

```
`file`
```

instead of:

```
'file'
```